worth2/templates/pagetree/page.html

```diff
@@ -106,8 +106,17 @@
 {% endif %}
 
 {% if next_section %}
-<li class="next"><a href="{{next_section.get_absolute_url}}">{{next_section.label}} &rarr;</a></li>
 
+{% comment %}
```

Comment:
@sdreher both good points. I will address unanswered quiz gating in a separate PR/PMT.

My implementation for video gating here doesn't take into account revisiting previously watched videos. I'll address that in this PR.